.nvmrc

```diff
@@ -0,0 +1 @@
+10.19.0
```

Comment:
This, `.ruby-version`, and `runtime.txt` help tell Netlify which versions of each respective runtime to use to build the website.